zcash_client_backend/src/data_api.rs

```diff
@@ -1182,5 +1189,12 @@ pub mod testing {
 
             Ok(())
         }
+
+        fn get_checkpoint_depth(
+            &self,
+            min_confirmations: NonZeroU32,
+        ) -> Result<usize, ShardTreeError<Self::Error>> {
+            Ok(usize::try_from(u32::from(min_confirmations) - 1).unwrap())
```

Comment:
Is the `- 1` here correct? I think `min_confirmations` is actually equal to the checkpoint depth, as they are both 1-indexed relative to the chain (`min_confirmations = 0` is "unmined, in the mempool").
```suggestion
            Ok(usize::try_from(u32::from(min_confirmations)).unwrap())
```

(The `- 1` in `get_checkpoint_depth` is correct, because there we're deriving an absolute height relative to the chain tip, so it is for adjusting from 1-indexing to 0-indexing.)